geometry.zsh

```diff
@@ -112,9 +117,29 @@ prompt_geometry_set_title() {
   prompt_geometry_print_title '%~'
 }
 
+# dumb hashing function
+prompt_geometry_hash_string() {
+  local sum=0
+  primes=(1 2 3 5 7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71 73 79 83 89 97 101 103 107 109 113 127 131 137 139 149 151 157 163 167 173 179 181 191)
+  ord_period=$(printf '%d' "'.")
+  for i in {1..${#1}}; do
+    letter=${1[$i]}
+    ord=$(printf '%d' "'$letter")
+    sum=$(($sum + $primes[$i] * $primes[1 + $(($ord - $ord_period))]))
+  done
+  echo $sum
+}
```

Comment:
I'm kinda conflicted on the hash function but I don't really think we'll need anything better than this.

Why are you using `$ord_period` in the calculation?